pkg/platform/defaults.go

```diff
@@ -325,7 +325,7 @@ func setPlatformDefaults(p *v1.IntegrationPlatform, verbose bool) error {
 	// Build timeout
 	if p.Status.Build.GetTimeout().Duration == 0 {
 		p.Status.Build.Timeout = &metav1.Duration{
-			Duration: 5 * time.Minute,
+			Duration: 10 * time.Minute,
```

Comment:
Maybe this is something we could just increase in the E2E tests by setting this duration on the IntegrationPlatform instances created by the E2E tests?

Reply:
Good one. This is changing the default existing behavior, not sure if it's something we should introduce so lightly.

Reply:
Done.